pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -246,21 +268,7 @@
       return segmentMetadata;
     } else {
       throw new ControllerApplicationException(LOGGER,
-          "Failed to find segment: " + segmentName + " in table: " + tableName, Status.NOT_FOUND);
-    }
-  }
-
-  @Nullable
```

Comment:
i could spot no change in this method vs same method moved at the bottom. can we keep it here, so that the scope of review stays limited to the actual changes?

Reply:
my bad.